lib/ansible/modules/files/file.py

```diff
@@ -196,6 +196,16 @@
     owner: foo
     group: foo
 
+- name: Remove file (delete file)
+  file:
+    path: /etc/foo
```

Comment:
I would give the example for removing a file some kind of extension, so it's clear that it's targeting a file rather than a directory. 
```suggestion
    path: /etc/foo.txt
```